src/view/components/codePreview.styl

```diff
@@ -16,7 +16,7 @@ governing permissions and limitations under the License.
   white-space: pre;
 }
 
-.CodePreview-openEditorButton {
+button.CodePreview-openEditorButton {
```

Comment:
Parcel v2 changed the order of some of the css files so that this was not overriding the spectrum style. Adding button makes this more specific so that it will override.